packages/rum-core/src/domain/rumEventsCollection/view/trackViews.ts

```diff
@@ -100,6 +111,11 @@ export function trackViews(
       startClocks,
       viewOptions
     )
+    activeViews.push(newlyCreatedView)
+    newlyCreatedView.stopObservable.subscribe(() => {
+      activeViews.splice(activeViews.indexOf(newlyCreatedView))
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Use a `Set` instead of an array for `activeViews`